napari/_qt/_tests/test_async_slicing.py

```diff
@@ -36,8 +36,9 @@ def test_async_slice_image_on_current_step_change(
 @pytest.mark.sync_only
 def test_async_slice_image_on_order_change(make_napari_viewer, qtbot, rng):
     viewer = make_napari_viewer()
-    data = rng.random((3, 4, 5))
+    data = rng.random((3, 5, 7))
```

Comment:
I made each dimension odd to ensure the middle slice is exactly defined regardless of rounding strategy. Then I also added the precondition check to avoid confusing error messages in the future.